api/gen/openapiv2/CampusService.swagger.json

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
Should we really check this in? (I'm not opposed)

Reply:
I wouldn't mind, most swagger tools accept urls as input so it could be nice to have it accessible via GitHub. 

Also, what do you think about compiling the Json to a github pages site pointing to docs.tum.app?

Reply:
Sounds good.